Configuration/DataProcessing/test/RunPromptReco.py

```diff
@@ -45,6 +43,8 @@ def __call__(self):
         if self.inputLFN == None:
             msg = "No --lfn specified"
             raise RuntimeError(msg)
+        if self.nThreads == None:
+            msg = "No --nThreads specified"
```

Comment:
@andresfelquintero I do not see the purpose of this statement, if no action or printout is added, the "msg" string is not used as it is now

Reply:
@andresfelquintero wait, in this case the comment of @davidlange6 is valid, you cannot have ```None``` as default value, but it should be 1, as in https://cmssdt.cern.ch/lxr/source/Configuration/Applications/python/ConfigBuilder.py#0087
otherwise you always force the user to specify explicitly the option

Reply:
so @slava77 do you think it is better to have a default value? or shall we let that input as a completely optional one?

Reply:
in configurations built with cmsDriver the parameter is optional

Reply:
If this is going to be an optional input for this code, then there is no point in having the error message or the if statement on line 46. Shall I delete it then?